src/flash.h

```diff
@@ -10,11 +10,8 @@
  * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
  */
 
-#ifndef SWIFTNAV_FLASH_CALLBACKS_H
-#define SWIFTNAV_FLASH_CALLBACKS_H
-
-#define FLASH_STM 0 /**< Value to pass flash callbacks to use STM Flash */
-#define FLASH_M25 1 /**< Value to pass flash callbacks to use M25 Flash */
+#ifndef SWIFTNAV_FLASH_H
+#define SWIFTNAV_FLASH_H
```

Comment:
This file is now just common return codes between the two kinds of flash? (Which is fine, just checking I understand)